tests/e2e/taqueria-plugin-flextesa.spec.ts

```diff
@@ -32,6 +32,39 @@ describe('E2E Testing for taqueria typechecker and simulator tasks of the tezos-
 		expect(accounts.stdout).toContain('bob');
 	});
 
+	test('Verify that taqueria can return JSON when request for list of accounts from a sandbox is made by TVsCE', async () => {
+		const accounts = await exec(`taq list accounts ${dockerName} --fromVsCode`, { cwd: `./${taqueriaProjectPath}` });
+		expect(accounts.stdout).toEqual(
+			JSON.stringify([
+				{
+					account: 'bob',
+					balance: '3000 ꜩ',
+					address: 'tz1aSkwEot3L2kmUvcoxzjMomb9mvBNuzFK6',
+				},
+				{
+					account: 'alice',
+					balance: '3000 ꜩ',
+					address: 'tz1VSUr8wwNhLAzempoch5d6hLRiTh8Cjcjb',
+				},
+				{
+					account: 'john',
+					balance: '3000 ꜩ',
+					address: 'tz1Zwoh1QCVAvJ4sVTojMp9pLYp6Ji4NoZy6',
+				},
+				{
+					account: 'jane',
+					balance: '3000 ꜩ',
+					address: 'tz1aHUAC4oviwJuZF1EvVSvFz7cu9KMNYBph',
+				},
+				{
+					account: 'joe',
+					balance: '3000 ꜩ',
+					address: 'tz1MVGjgD1YtAPwohsSfk8i3ZiT1yEGM2YXB',
+				},
+			]) + '\n',
```

Comment:
@mweichert Do we have `tests` explicitly excluded from the auto formatting or something is wrong with the config? 😕

Reply:
We do explicitly disable the formatting of tests data.

We were having issues with our strings being reformatted here:
~/Projects/taqueria/tests/e2e/data/help-contents.ts